bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/service.js

```diff
@@ -0,0 +1,58 @@
+import { makeCall } from '/imports/ui/services/api';
+import Storage from '/imports/ui/services/storage/session';
+import Auth from '/imports/ui/services/auth';
+
+const sendAnnotation = (annotation) => {
+  makeCall('sendAnnotation', annotation);
+};
+
+const getWhiteboardToolbarValues = () => {
+  const drawSettings = Storage.getItem('drawSettings');
+  if (drawSettings) {
+    const {
+      whiteboardAnnotationTool,
+      whiteboardAnnotationThickness,
+      whiteboardAnnotationColor,
+      textFontSize,
+      textShape,
+    } = drawSettings;
+
+    return {
+      tool: whiteboardAnnotationTool,
+      thickness: whiteboardAnnotationThickness,
+      color: whiteboardAnnotationColor,
+      textFontSize,
+      textShapeValue: textShape.textShapeValue ? textShape.textShapeValue : '',
+      textShapeActiveId: textShape.textShapeActiveId ? textShape.textShapeActiveId : '',
+    };
+  }
+  return undefined;
```

Comment:
Shouldn't you return a empty object in this case for a more consistence return?

And you can invert this flow to:

```
if(!drawSettings){
 return {}
}

const{
    whiteboardAnnotationTool,
...............
...................
```